packages/pcset/index.d.ts

```diff
@@ -0,0 +1,127 @@
+type Binary = "0" | "1";
+type Chroma = Binary[12];
+type Interval = string;
+/**
+ * Get chroma of a pitch class set. A chroma identifies each set uniquely.
+ * It"s a 12-digit binary each presenting one semitone of the octave.
+ *
+ * Note that this function accepts a chroma as parameter and return it
+ * without modification.
+ *
+ * @param {Array|String} set - the pitch class set
+ * @return {String} a binary representation of the pitch class set
+ * @example
+ * PcSet.chroma(["C", "D", "E"]) // => "1010100000000"
+ */
+export declare function chroma(set: string | string[]): Chroma;
+/**
+ * Get a list of all possible chromas (all possible scales)
+ * More information: http://allthescales.org/
+ * @return {Array} an array of possible chromas from '10000000000' to '11111111111'
+ *
+ */
+export declare function chromas(numberOfNotes: number): Chroma[];
```

Comment:
I'm afraid is this way (see previous comment):

```suggestion
export declare function chromas(numberOfNotes: number): string;
```